.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow will deploy the application to the running environment after a successful merge to main
+
+name: deploy
```

Comment:
This attempts to deploy a flask application with the name flaskbb you team didn't use flask to build the application and that is not the name of the application you created.